tools/scripts/brokenLinkChecker/brokenLinkChecker.json

```diff
@@ -90,12 +100,15 @@
     "http://www.usnews.com/news/stem-solutions/articles/2014/11/25/making-it-count-computer-science-spreads-as-graduation-requirement",
     "http://www.usnews.com/opinion/knowledge-bank/articles/2016-02-11/arkansas-and-washington-lead-efforts-to-expand-computer-science-education",
     "http://www.wsj.com/articles/why-coding-is-your-childs-key-to-unlocking-the-future-1430080118",
+    "https://amara.org/en/videos/",
     "https://applitools.com/",
+    "https://anitab.org/",
```

Comment:
haven't seen this before :) why do we decide these links are to be ignored? For example this link works.

Reply:
Oh, these links should all work in a real browser, but for some reason our broken link checker isn't able to validate them, and so we add them to the ignore list here.